samples/hybrid-connections/node/hyco-websocket-tunnel/lib/tunnel.js

```diff
@@ -78,7 +78,8 @@ exports.createTunnel = function(wsServerAddr, token, credentials, listen, forwar
       tcpSock.destroy();
     });
 
-    var url = wsServerAddr; //+ '&port=' + forward.port + '&host=' + forward.host;
+    //var url = wsServerAddr; //+ '&port=' + forward.port + '&host=' + forward.host;
```

Comment:
>//var url = wsServerAddr; //+ '&port=' + forward.port + '&host=' + forward.host; [](start = 4, length = 80)

It would be nice to remove the commented out code it code doing the same thing gets put back in.